master/getting-started/kubernetes/upgrade/test.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Testing the data migration
+no_canonical: true
+---
+
+1. Run `calico-upgrade dry-run` to validate the migration.
+
+   > **Note**: By default, `calico-upgrade` creates a directory called `calico-upgrade-report` 
+   > in the current working directory to store its report files. If you want it to write
+   > its files to a different location, include a `--output-dir` flag in your command and 
+   > specify the alternate path.
+   {: .alert .alert-info}
```

Comment:
Something that confused me for a while is that `calico-upgrade dry-run --output-dir=MY-DIR` works, but `calico-upgrade --output-dir=MY-DIR dry-run` does not.  In other words, the `--output-dir` bit has to come *after* the `dry-run` command.  Is that something that you'd want to cover in these docs?  (Unless this was a bug that has since been fixed....)

Reply:
Thanks for the info about this! I will add this.